README.md

```diff
@@ -76,6 +76,8 @@ No resources.
 | <a name="input_region"></a> [region](#input\_region) | The region where resources will be created | `string` | `"us-central1"` | no |
 | <a name="input_use_local_chart"></a> [use\_local\_chart](#input\_use\_local\_chart) | Whether to use a local chart instead of one from a repository | `bool` | `false` | no |
 | <a name="input_use_self_signed_cluster_issuer"></a> [use\_self\_signed\_cluster\_issuer](#input\_use\_self\_signed\_cluster\_issuer) | Whether to install and use a self-signed ClusterIssuer for TLS. Due to limitations in Terraform, this may not be enabled before the cert-manager CRDs are installed. | `bool` | `false` | no |
+| <a name="input_storage_bucket_versioning"></a> [use\_storage\_bucket\_versioning](#input\_use\_storage\_bucket\_versioning) | Enable bucket versioning (Disabled by default due to deletion timeouts). | `bool` | `false` | no |
```

Comment:
? We probably should regenerate the readme.md using terraform-doc?  since I think it's just `storage_bucket_versioning` and `storage_bucket_version_ttl` instead of `use_storage ...`